Update email configuration tips from troubleshooting article (#22)
kobotoolbox/kobotoolbox.github.io: docs/_articles/kpi-frontend-development-troubleshooting.md

### Email-activating fake account

> Sometimes you create a test account on local environment with just a username and password (no email address). If you try to log in, you will see "confirm email" message. You don't need to create temporary email addresses, you can simply add one line on your dev settings (for `kobo-install` it's most probably `kobo-env/envfiles/kpi.txt`; `kobo-env` is a directory that is created as a sibling to `kobo-install`):

Reviewer: ```suggestion
Sometimes you create a test account on local environment with just a username and password (no email address). If you try to log in, you will see "confirm email" message. You don't need to create temporary email addresses, you can simply add one line on your dev settings (for `kobo-install` it's `kobo-env/envfiles/kpi.txt`; `kobo-env` directory is a sibling to `kobo-docker` configured path):
```

```
EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

> After that you will see the email content in your logs (so either `./run.py --logs` or through docker). You can click the activation link through there.

Reviewer: ```suggestion
After that you will see the email content in your logs (e.g. `./run.py -cf logs -f kpi`) for both email activation and password reset emails. You can click the link from the logs.
```
